sdk/src/main/java/com/silanis/esl/sdk/examples/PackageReferencedConditionsExample.java

```diff
@@ -0,0 +1,139 @@
+package com.silanis.esl.sdk.examples;
+
+import static com.silanis.esl.sdk.builder.DocumentBuilder.newDocumentWithName;
+import static com.silanis.esl.sdk.builder.PackageBuilder.newPackageNamed;
+import static com.silanis.esl.sdk.builder.SignatureBuilder.signatureFor;
+import static com.silanis.esl.sdk.builder.SignerBuilder.newSignerWithEmail;
+import static org.joda.time.DateTime.now;
+
+import com.silanis.esl.sdk.DocumentPackage;
+import com.silanis.esl.sdk.DocumentType;
+import com.silanis.esl.sdk.FieldCondition;
+import com.silanis.esl.sdk.FieldId;
+import com.silanis.esl.sdk.ReferencedConditions;
+import com.silanis.esl.sdk.SignatureId;
+import com.silanis.esl.sdk.builder.FieldBuilder;
+
+/**
+ * Example SDK code for the /referencedConditions
+ *
+ * @author x_MacieMi1 (Michał Maciejewski)
+ */
+public class PackageReferencedConditionsExample extends SDKSample {
+
+    private static final String DOCUMENT_ID_1 = "documentId1";
+    private static final String DOCUMENT_ID_2 = "documentId2";
+
+    private final SignatureId signatureId1 = new SignatureId("signatureId1");
+    private final SignatureId signatureId2 = new SignatureId("signatureId2");
+
+    private final FieldId fieldId1 = new FieldId("fieldId1");
+    private final FieldId fieldId2 = new FieldId("fieldId2");
+    private final FieldId fieldId3 = new FieldId("fieldId3");
+    private final FieldId fieldId4 = new FieldId("fieldId4");
+    private final FieldId fieldId5 = new FieldId("fieldId5");
+    private final FieldId fieldId6 = new FieldId("fieldId6");
+
+    private ReferencedConditions packageLevelRefConditions;
+    private ReferencedConditions documentLevelRefConditions;
+    private ReferencedConditions fieldLevelRefConditions;
```

Comment:
This is an example. Define the above 3 fields as public and remove the private methods.

Reply:
Updated